Sources/SwiftFormatCore/SyntaxLintRule.swift

```diff
@@ -38,9 +38,15 @@ extension Rule {
     on node: Syntax?,
     actions: ((inout Diagnostic.Builder) -> Void)? = nil
   ) {
+    let loc = node.map {
```

Comment:
Can you add a TODO comment here for us to investigate why `node?.startLocation(in: context.fileURL)` isn't working, since the default argument in that function says we should be skipping the trivia already?